test/e2e/opaque_resource.go

```diff
@@ -59,11 +59,19 @@ var _ = framework.KubeDescribe("Opaque resources [Feature:OpaqueResources]", fun
 			}
 		}
 
-		removeOpaqueResource(f, node.Name, opaqueResName)
 		addOpaqueResource(f, node.Name, opaqueResName)
 	})
 
+	// TODO: The suite times out if removeOpaqueResource is called as part of
```

Comment:
So I looked at anti-affinity and it does both, so it looks like they just patched and didn't bother to clearly define a happy path.